wcfsetup/install/files/ts/WoltLabSuite/Core/Event/Handler.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * Versatile event system similar to the WCF-PHP counter part.
+ *
+ * @author  Alexander Ebert
+ * @copyright  2001-2019 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  EventHandler (alias)
+ * @module  WoltLabSuite/Core/Event/Handler
+ */
+
+import * as Core from '../Core';
+import Devtools from '../Devtools';
+
+const _listeners = new Map<string, Map<string, Map<string, Callback>>>();
```

Comment:
```suggestion
type Identifier = string;
type Action = string;
type Uuid = string;
const _listeners = new Map<Identifier, Map<Action, Map<Uuid, Callback>>>();
```

And then consistent adjustments for the rest.